src/manifold/src/quickhull.cpp

```diff
@@ -1,3 +1,22 @@
+// Copyright 2024 The Manifold Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+ * Implementation of the 3D QuickHull algorithm by Antti Kuukka
+ *
+ * No copyrights. What follows is 100% Public Domain.
```

Comment:
Not sure if this is the best way to put it, I think we typically just acknowledge the source, put an URL as well as state its license. From my understanding, the code is under a certain license doesn't mean the author has no copyright over it.

Reply:
This was what he had included in his code. I can include the link, but I don't know which license to acknowledge, could you just tell me what exactly do I need to include I will add that below our license.

Reply:
Public domain *does* mean there is no longer a copyright holder. We should modify this line, since our version is no longer public domain (see the apache license above). I would change this to "Derived from the public domain work of Antti Kuukka at github/..."

Reply:
I've made the change to the license, I hope it's alright now.